packaging/conf/internalsso-openidc.conf.in

```diff
@@ -0,0 +1,48 @@
+LoadModule auth_openidc_module modules/mod_auth_openidc.so
+
+ServerName @ENGINE_FQDN@
```

Comment:
Hmm, are you sure about it? Shouldn't the ServerName be defined only once as a part of ovirt-engine setup?

Reply:
Well, I have not put too much consideration into it - followed the original instructions from the very beginning. You actually might be right about it - need to check this out. Thanks!

Reply:
Removed